Brooklyn/Contexts/BrooklynManager.swift

```diff
@@ -70,4 +70,14 @@ extension BrooklynManager {
         hasRandomOrder.toggle()
         Database.standard.set(hasRandomOrder: hasRandomOrder)
     }
+    
+    var currentVersion: String {
+        let appVersion = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String
+        let buildNumber = Bundle.main.infoDictionary?["CFBundleVersion"] as? String
+        var versionString = appVersion ?? ""
+        if let buildNumber = buildNumber {
+            versionString.append(" (\(buildNumber))")
+        }
+        return versionString
+    }
```

Comment:
Some little things:
- Let's make this a `let` since it's not going to change when the screensaver preferences Is open
- We kind of ignore the build number in Brooklyn, so let's just discard and only show the version

> However, if I built the bundle itself and added that to my own preferences on my Mac, it didn't give me version 2.0.1 (3) as I would have expected. Instead it gave me version 14.0
- Instead of doing `Bundle.main`, specifying its identifier should solve this issue

In short, you could update this to:
```swift
enum Constant {
    static let bundleId = "oedrommcarrasco.brooklyn"
    static let versionKey = "CFBundleShortVersionString"
}

let currentVersion = Bundle(identifier: Constant.bundleId)?.infoDictionary?[Constant.versionKey] as? String ?? ""
```

p.s.: Yes, I'm aware the bundle id should be `pedrommcarrasco.brooklyn` instead of `oedrommcarrasco.brooklyn` but well, mistakes were made and now it's a little too late to change 😓

Reply:
> Let's make this a let since it's not going to change when the screensaver preferences Is open

computed properties can't be `let`s, if I try, the compiler complains saying "'let' declarations cannot be computed properties".

It is however not possible to mutate the value, as I have not declared a `set` method.

Let me know if this is OK with you, otherwise I can change it into a `function`.

Reply:
Oh...and btw, changing the bundle-id did the trick, thanks 👍 

I took the liberty of checking if we get a value back from `Bundle(identifier:)`, just so we can populate the button with a value in the `Canvas` example as well. Here's how it looks:

```
var currentVersion: String {
    //We are running from the Brooklyn screensaver itself
    if let screensaverBundle = Bundle(identifier: Constant.bundleId) {
        return screensaverBundle.infoDictionary?[Constant.versionKey] as? String ?? ""
    } else {
        //we are running from canvas, lets return the version number from there...just to return something
        return Bundle.main.infoDictionary?[Constant.versionKey] as? String ?? ""
    }
}
```

Reply:
I didn't mean to make it a computed property and a `let`.
For example, this `let currentVersion = Bundle(identifier: Constant.bundleId)?.infoDictionary?[Constant.versionKey] as? String ?? ""` would work but it's fine, I like your changes!


> It is however not possible to mutate the value, as I have not declared a set method.

My problem wasn't with it being mutated but every time it was accessed it was computed without being needed. In order to solve this and since with your current implementation `let` isn't a solution anymore, we could make it a `lazy var`

Also, I know I'm nitpicking, but please add a space between `//` and the actual comment 😅 💅

Reply:
Ah, right, gotcha 👍 

I'll change to `lazy var` and add spaces in the comments.